bfsaz/bfsaz.go

```diff
@@ -1,443 +0,0 @@
-// Package bfsaz abstracts Azure Blob Storage.
```

Comment:
Uhh, what if someone/we need that again?

Reply:
It's based on a VERY old az SDK, we would have to write it from scratch anyway